### PR TITLE
update README for new Babel 6.13

### PR DESCRIPTION
**Pull request by zeitiger**

### CODE REVIEW

**Comment by zeitiger** [comment]
no Travis I don't believe that I break anything we my markdown change 😼

File: README.md [hunk 1]
```diff
@@ -56,22 +56,30 @@ Use `babelrc: false` to prevent Babel from using local (i.e. to your external de
...
+  "plugins": [
+    "external-helpers"
+  ]
 }
 ```
 
 ### Modules
 
-The `es2015` preset includes the [transform-es2015-modules-commonjs](http://babeljs.io/docs/plugins/transform-es2015-modules-commonjs/) plugin, which converts ES6 modules to CommonJS – preventing Rollup from working. Instead, you should either use the `es2015-rollup` preset, which excludes that plugin, or otherwise ensure that the `modules-commonjs` plugin is excluded. Rollup will throw an error if this is incorrectly configured.
+The `es2015` preset includes the [transform-es2015-modules-commonjs](http://babeljs.io/docs/plugins/transform-es2015-modules-commonjs/) plugin, which converts ES6 modules to CommonJS – preventing Rollup from working. Since Babel 6.3 it's possible to deactivate module transformation with `"modules": false`. So there is no need to use the old workaround with `babel-preset-es2015-rollup`, that work for Babel <6.3. Rollup will throw an error if this is incorrectly configured.
```
Comment (eventualbuddha) [inline README.md]: I believe the version of babel that changed it is [v6.13.0](https://github.com/babel/babel/releases/tag/v6.13.0).

**Comment by kokarn** [comment]
This really needs to be merged because it doesn't seem to work on new babel versions without this change.

**Comment by eventualbuddha** [comment]
Actually, both approaches work. But I prefer the updated one so merging.

**Pull request merged**